node_modules/gh-groups/lib/rest.js

```diff
@@ -23,6 +23,32 @@ var GrassHopper = require('gh-core');
 var GroupsAPI = require('./api');
 
 /**
+ * @REST getUserMemberships
+ *
+ * Get the groups a user is a member of
+ *
+ * @Server      admin,app
+ * @Method      GET
+ * @Path        /users/{id}/memberships
+ * @PathParam   {number}            id                      The id of the user to get the upcoming events for
+ * @QueryParam  {number}            [limit]                 The maximum number of results to retrieve. Default: 25
+ * @QueryParam  {number}            [offset]                The paging number of the results to retrieve
+ * @Return      {EventList}                                 The upcoming events for the user
+ */
+var getUserMemberships = function(req, res) {
```

Comment:
We seem to have not followed this pattern for our `rest.js` files. If we're going to change, I'd like to do them all in one big sweep at that point. Is that OK?